src/main/java/duke/commands/Event.java

```diff
@@ -0,0 +1,16 @@
+package duke.commands;
+
+public class Event extends Task{
+
+    protected String eventDate;
```

Comment:
Is there a reason why this field is protected instead of private?

Reply:
Actually it does make more sense to be private instead of protected, I'll change that